mllib/src/main/scala/org/apache/spark/mllib/linalg/distributed/RowMatrix.scala

```diff
@@ -775,6 +778,28 @@ class RowMatrix @Since("1.0.0") (
         s"The number of rows $m is different from what specified or previously computed: ${nRows}.")
     }
   }
+
+  /**
+   * Computing desired tree aggregate depth necessary to avoid exceeding
+   * driver.MaxResultSize during aggregation.
+   * Based on the formulae: (numPartitions)^(1/depth) * objectSize <= DriverMaxResultSize
+   * @param aggregatedObjectSizeInBytes the size, in megabytes, of the object being tree aggregated
+   */
+  private[spark] def getTreeAggregateIdealDepth(aggregatedObjectSizeInBytes: Long) = {
+    val maxDriverResultSizeInBytes = rows.conf.get[Long](MAX_RESULT_SIZE)
```

Comment:
Can you use getLong here?

Reply:
I don't know about that...
I thought the aim of MAX_RESULT_SIZE was precisely to avoid having to call getLong with hardcoded parameter string "spark.driver.maxResultSize", and to guarantee that the number returned will be in Bytes unit...
Did I get it wrong ?

Reply:
Oh right, maybe I really mean: can you remove `[Long]`? but I guess you're using that to cast, so it's fine